src/OpenLoco/src/Windows/LandscapeGeneration.cpp

```diff
@@ -1068,8 +1068,11 @@ namespace OpenLoco::Ui::Windows::LandscapeGeneration
 
                 case widx::number_of_towns_down:
                 {
-                    uint16_t newNumTowns = std::max<uint16_t>(Limits::kMinTowns, options.numberOfTowns - 1);
-                    options.numberOfTowns = newNumTowns;
+                    if (options.numberOfTowns > 0)
+                    {
+                        uint16_t newNumTowns = std::max<uint16_t>(Limits::kMinTowns - 1, options.numberOfTowns - 1);
```

Comment:
could also change kMinTowns to 0 since this is the only use-case, but I think this is more appropriate to leave it at 1, and subtract the 1 here.

Reply:
Will add a comment. The editor step validation works, in that it's currently identical to vanilla as of this branch. Unsure if it's RE'd yet though

Reply:
@duncanspumpkin Had a peek, existing code checks for `empty()` - should we change it to compare to kMinTowns?

https://github.com/OpenLoco/OpenLoco/blob/5eab78d6b3adeb2401679e81f88f5e0f17b3c3c1/src/OpenLoco/src/EditorController.cpp#L106C9-L111C75

Reply:
Yeah we might as well.

Reply:
done